lib/dashboard.js

```diff
@@ -52,7 +52,7 @@ Dashboard.prototype._configureKeys = function () {
   // this key will be watched on the global screen
   this.screen.ignoreLocked = ["C-c"];
   this.screen.key("C-c", () => {
-    process.exit(0); // eslint-disable-line no-process-exit
+    process.kill(process.pid, "SIGINT");
```

Comment:
Yep this looks right! The only thing we did different in `webpack-dashboard` was listen for `"escape"` and `"q"` as valid keys to trigger sending `SIGINT`. We could consider doing that here as well.

Reply:
Good idea! Added https://github.com/FormidableLabs/nodejs-dashboard/issues/106 to allow some discussion first.